src/test/socket/connect/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_test(NAME connect COMMAND ../../target/debug/test_connect)
+foreach(Method ptrace preload)
+    # runs the shadow tests with only the passing tests, not all tests
+    add_test(NAME connect-shadow-${Method} COMMAND ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=${Method}
+	    -l debug -d connect.shadow-${Method}.data ${CMAKE_CURRENT_SOURCE_DIR}/connect.shadow.config.xml)
+endforeach()
+
+set(TestNames connect connect-shadow-ptrace connect-shadow-preload)
```

Comment:
Consider appending to TestNames next to corresponding add_test commands, to make it easier to keep it consistent with the actual set of tests.

Reply:
Added.